tools/src/h5diff/h5diff_common.c

```diff
@@ -56,6 +56,28 @@ static struct h5_long_options l_opts[] = {{"help", no_arg, 'h'},
                                           {"vfd-info-2", require_arg, 'Z'},
                                           {NULL, 0, '\0'}};
 
+static H5FD_onion_fapl_info_t onion_fa_g_1 = {
```

Comment:
Comment for future reference:

The tools should have basically no idea about the Onion VFD or its structure definitions, but for now this is good enough to make the tools play nice.

Reply:
Agree. Aren't we waiting on some library fixes, though? We can make the changes after the merge.